ball.js

```diff
@@ -0,0 +1,147 @@
+//https://maththebeautiful.com/ball-bounce/
+
```

Comment:
The name of the file must be Ball.js - It is not mandatory now but a good practice